### DISCUSSION TITLE
Flightplan with WCA and TRACK

### DISCUSSION THREAD

**Discussion by Lewis2023** [albar965/littlenavmap#1044] [Ideas]
Hi
I would prefer to have the terms Wind Correction Angle and Track directly in the Flightplan.
This values are calculated and shown within the compass rose, but why not in the Flightplan?
Thanks
Ed

**Comment by albar965** [comment]
WCA and track depend on wind and the flight plan table is static. This applies to fuel too. 
The values are already shown in the aircraft progress tab.

Kind of duplicate of #371
